lib/sequence.js

```diff
@@ -13,15 +13,15 @@ class Sequence extends Readable {
     this.runPipelines().catch(err => this.emit('error', err))
   }
 
-  runPipelines () {
-    return this.pipelines.reduce((promise, pipeline) => {
-      return promise.then(() => {
-        const next = eventToPromise(pipeline, 'end')
-        pipeline.read()
-        return promise.then(next)
-      })
-    }, Promise.resolve())
-  };
+  async runPipelines () {
+    for (const pipeline of this.pipelines) {
+      pipeline.resume()
```

Comment:
Is that method of built-in stream API?

Reply:
Yes, [Readable.resume](https://nodejs.org/api/stream.html#stream_readable_resume), it also has the benefit of consuming all chunks and not just a single one. Was actually not a problem with the pipeline like it is now, but when I was testing and writing chunks in the sub pipelines, it didn't reach the end. Btw. the base package should get an [operation](https://github.com/zazuko/barnard59-main/issues/16) `concat`, which could also handle this case.